MetaMask/metamask-extension
# UX Multichain: updated border top for activity list

With the ListItem as the parent, the border-top in the transaction list item was not removed. This PR fixes that

I have rendered the style conditionally since this component is not using Box internally, once we update the TransactionListItem for Multichain work, I will remove this inline styling.

Fixes #19142 

## Screenshots/Screencaps

## Non-Multichain 

### Before


https://github.com/MetaMask/metamask-extension/assets/39872794/784187f5-e1fb-4618-9e66-1adccdb50052


### After

https://github.com/MetaMask/metamask-extension/assets/39872794/3f1a7282-aed2-4346-8512-313990323f78


## With Multichain

### Before


https://github.com/MetaMask/metamask-extension/assets/39872794/a588f909-1582-4948-97dd-20e7ee8410c7


### After


https://github.com/MetaMask/metamask-extension/assets/39872794/e575ec98-c419-487e-9dfb-369dfd54561c


## Manual Testing Steps

- Go to this home screen
- Check there is no border after Assets, Activity and NFTs Tab heading in MULTICHAIN Work


## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
